### ISSUE TITLE
Bug: jsx-a11y throws linting errors ("Definition for rule '[rule]' was not found") for all rules 

### ISSUE THREAD

**Issue by dominicchapman** [tambium/web-foundations#1]
When including `plugin:@tambium/react`, [`jsx-a11y`](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/) (used within our react plugin) causes linting errors of the following form at the start of all files being linted:

> Definition for rule 'accessible-emoji' was not found

This is thrown for every rule we include in `lib/config/rules/jsx-a11y` and shows up in a number of issues on the repository for the `jsx-a11y` plugin – for example, [here](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/issues/262).

The suggestion in all cases seems to be use of `npm ls` to identify which dependencies are missing but I’m not convinced this is suitable for this repository given that we use yarn workspaces. 

For the meantime, I have [removed the rules](https://github.com/tambium/web-foundations/commit/f71af8ff3ab66d7e888830974b954470128b560c) but this should be revisited to provide guidance on accessibility in projects.
